service/instance.py

```diff
@@ -235,7 +235,7 @@ def remove_ips(esh_driver, esh_instance, identity_uuid, update_meta=True):
     Returns: (floating_removed, fixed_removed)
     """
     from service.tasks.driver import update_metadata
-    core_identity = Identity.objects.get(uuid=core_identity_uuid)
+    core_identity = CoreIdentity.objects.get(uuid=identity_uuid)
```

Comment:
Here the Identity model was imported as CoreIdentity, and the identity_uuid was passed in as a parameter.